packages/node_modules/@webex/internal-plugin-wdm/src/interceptors/url.js

```diff
@@ -31,6 +31,10 @@ export default class UrlInterceptor extends Interceptor {
       return this.webex.internal.device.getServiceUrl(options.service)
         .then((uri) => {
           if (!uri) {
+            if (options.service === 'web-authentication') {
```

Comment:
this feels hacky and brittle. suggestions, @InteractiveTimmy ?

Reply:
I don't understand why this is needed. If we've fetched the proximity based catalog on initialization, wouldn't it be listed in the services interceptor? Or does the request happen before it's completely initialized?

Reply:
@andrewholsted, this change is needed [for now] since that service does not exist in the catalog returned from **WDM** catalog, which is what this interceptor is using. There is another interceptor very similar to this that is used by the services plugin.

Reply:
@robstax, a test to validate that this works would be useful, you could append the test here: `packages/node_modules/@webex/internal-plugin-wdm/test/unit/spec/interceptors/device-url.js L29`.